### PR TITLE
docs: add documentation for AudioReader

### PR DESCRIPTION
**Pull request by theUnkownName**
This PR is for [JINAHUBV1-359](https://jinaai.atlassian.net/browse/JINAHUBV1-359?atlOrigin=eyJpIjoiYTJhY2M5ZWEwZjMwNGViNTk0YjUxMDZhMmRlNmU4MjAiLCJwIjoiaiJ9).

In the Manifest file:

- [x] short description
- [x] points to the README (URL) of the respective Image
- [x] Correct keywords

In the README:

- [x] 1-2 sentence description of the "Executor" and the functionality of it
- [x] Example Section with code snipped of  "how to use it in flow"

### CODE REVIEW

**Review by cristianmtr — DISMISSED** [review]

File: crafters/audio/AudioReader/README.md [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+   ```bash
+    docker run jinahub/pod.crafter.audioreader:0.0.8-1.0.1 --port-in 55555 --port-out 55556
+    ```
+    
+2. Run with Flow API
+   ```python
+    from jina.flow import Flow
+
+    f = (Flow()
+        .add(name='my-encoder', image='jinahub/pod.crafter.audioreader:0.0.8-1.0.1', port_in=55555, port_out=55556))
```
Comment (Yongxuanzhang) [inline crafters/audio/AudioReader/README.md:29]:
```suggestion
        .add(name='my_encoder', uses='docker://jinahub/pod.crafter.audioreader:0.0.8-1.0.1', port_in=55555, port_out=55556))
```

File: crafters/audio/AudioReader/README.md [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+    ```
+    
+3. Run with Jina CLI
+   ```bash
+    jina pod --uses jinahub/pod.crafter.audioreader:0.0.8-1.0.1 --port-in 55555 --port-out 55556
```
Comment (Yongxuanzhang) [inline crafters/audio/AudioReader/README.md:34]:
```suggestion
    jina pod --uses docker://jinahub/pod.crafter.audioreader:0.0.8-1.0.1 --port-in 55555 --port-out 55556
```

**Review by Yongxuanzhang — CHANGES_REQUESTED** [review]

File: crafters/audio/AudioReader/README.md (hunk 1 above)
Comment (theUnkownName) [reply crafters/audio/AudioReader/README.md:29]: We also need "docker" here? it's not specified in our docs, but if we do need it, we also need to change the docs https://docs.jina.ai/chapters/hub/build-pod-image.html

Comment (Yongxuanzhang) [reply crafters/audio/AudioReader/README.md:29]: This doc is not updated. Same with this [one](https://github.com/jina-ai/docs/blob/ed9472013ddf43370dde4669be5f0f0379053e65/chapters/hub/use-your-pod.md).
the 'image='image_name' is not outdated and we should use "uses='docker://image_name" instead  of that one. And if you run this locally. The first error will be the my-encode name, the second issue is it won't use the image, although there are no errors.

```
f = (Flow()
        .add(name='my-encoder', image='jinahub/pod.crafter.audioreader:0.0.8-1.0.1', port_in=55555, port_out=55556)) 

```

**Review by Yongxuanzhang — APPROVED** [review]

**Pull request merged**